src/test_workflow/integ_test/integ_test_suite_opensearch_dashboards.py

```diff
@@ -48,6 +55,12 @@ def __init__(
             bundle_manifest_opensearch,
             build_manifest_opensearch
         )
+        self.repo = GitRepository(
+            build_manifest_opensearch_dashboards.components["functionalTestDashboards"].repository,
```

Comment:
Nit: Adding some comments letting people know why we hardcode functionalTestDashboards.

Reply:
Done Peter

Reply:
Thanks.